packages/trader/src/App/app.tsx

```diff
@@ -30,14 +31,16 @@ const App = ({ passthrough }: Apptypes) => {
 
     return (
         <MobxContentProvider store={root_store}>
-            <React.Fragment>
-                <Routes />
-                <TradeModals />
-                <NetworkStatusToastErrorPopup />
-                <TradeHeaderExtensions store={root_store} />
-                <TradeFooterExtensions />
-                <TradeSettingsExtensions store={root_store} />
-            </React.Fragment>
+            <StoreProvider store={root_store}>
+                <React.Fragment>
```

Comment:
@hamza-deriv The fragment seems redundant 🤔

Reply:
Its already there so can ignore for now